source/module.c

```diff
@@ -237,6 +298,105 @@ int aws_napi_callback_clean_up(struct aws_napi_callback *cb) {
     return AWS_OP_SUCCESS;
 }
 
+static void s_handle_failed_callback(napi_env env, napi_status status) {
```

Comment:
`status` gets passed in, but it's immediately overwritten by `if ((status = napi_xyz()))` calls
is that on purpose?

Reply:
Yes. The status parameter is literally only there to be the initial value of pending_exception, which is in turn only set to that in case the next call fails and you're debugging. Otherwise, it's all meaningless.